src/domUpdates.js

```diff
@@ -28,19 +30,68 @@ let domUpdates = {
           `
       )
     })
-    // getFavorites();
   },
-  cardButtonConditionals(event) {
-    console.log(event.target)
-    if (event.target.classList.contains('favorite')) {
-      favoriteCard(event);
-    } else if (event.target.classList.contains('card-picture')) {
-      displayDirections(event);
-    } else if (event.target.classList.contains('home')) {
+  viewFavorites(cardArea, user, favButton, currentFavs,) {
+    if (!user.favoriteRecipes.length) {
+      favButton.innerHTML = 'You have no favorites!';
+      return
+    } else {
+      favButton.innerHTML = 'Refresh Favorites'
+      cardArea.innerHTML = '';
+      this.populateCards(user.favoriteRecipes, cardArea, currentFavs)
+    }
+  },
+  displayFavorite(event, favButton) {
+    if (!event.target.classList.contains('favorite-active')) {
+      event.target.classList.add('favorite-active');
       favButton.innerHTML = 'View Favorites';
-      populateCards(cookbook.recipes);
+      return  true
+    } else if (event.target.classList.contains('favorite-active')) {
+      event.target.classList.remove('favorite-active');
+      return false
```

Comment:
This function now serves to just check if the function is active or not, and return the correct boolean value